vgstation13.dme

```diff
@@ -2282,7 +2282,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```

Comment:
👏 real 👏  coder 👏 hours 👏

Reply:
you know what's funny? I'm the one who added this travis check and I'm the only one who keeps forgetting about it